tests/test_generate_data.py

```diff
@@ -313,8 +313,10 @@ def test_generate(self):
             generate_data(
                 client=MagicMock(),
                 logger=mocked_logger,
-                model_family="merlinite",
-                model_name="models/merlinite-7b-lab-Q4_K_M.gguf",
+                model_family="granite",
+                model_name=os.path.join(
+                    TEST_DATA_DIR, "models/instructlab/granite-7b-lab"
+                ),
```

Comment:
Why do we need to change the model family and name here? We don't use granite as a teacher model, so it feels odd to use it here in the test.

Reply:
That's reasonable - thanks for bearing with all my questions!